### PR TITLE
[AS-85] Polygon can be created

### PR DESCRIPTION
**Pull request by mslup**

### CODE REVIEW

File: src/feature/3dutils/Model.tsx
Comment (andrmdmd) [inline src/feature/3dutils/Model.tsx:1]: folder names should follow camel case as we decided together

File: src/feature/3dutils/Sphere.tsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+export function Sphere({
+  position,
+  type,
+} :
+   {position: Point2D, type: SphereType}): JSX.Element {
+  return (
+    <mesh position={[position.x, 0, position.y]}>
+      <sphereGeometry args={[0.1, 32, 32]} />
+      {(() => {
```
Comment (andrmdmd) [inline src/feature/3dutils/Sphere.tsx]: you can define this function in the Sphere function body and only run it from here

File: src/redux/slices/creator.tsx [hunk 2]
```diff
@@ -27,6 +46,7 @@ const creatorSlice = createSlice({
       const x = round(action.payload.x, 2);
       const y = round(action.payload.y, 2);
       state.points = [...state.points, { x, y }];
+      console.log(state.points);
```
Comment (andrmdmd) [inline src/redux/slices/creator.tsx]: leftover debug

File: src/redux/slices/creator.tsx [hunk 3]
```diff
@@ -6,17 +6,36 @@ import { RootState, ThunkActionVoid } from '..';
...
 }
 
+interface ChangeInteractionPayload {
+  interaction: Interaction;
+}
+
 export const initialState: CreatorState = {
-  points: [],
+  points: [
+    // { x: -2, y: -2 },
```
Comment (andrmdmd) [inline src/redux/slices/creator.tsx]: leftover debug

**Comment by andrmdmd** [comment]
GJ! Is the grid getting thicker in the room (`/editor`) editor also expected?

File: src/feature/3dutils/Model.tsx
Comment (mslup) [reply src/feature/3dutils/Model.tsx:1]: yes!

**Comment by mslup** [comment]
> GJ! Is the grid getting thicker in the room (`/editor`) editor also expected?

thank you:) yes:)

**Review by andrmdmd — APPROVED** [review]

**Pull request merged**